eng/native/configurecompiler.cmake

```
@@ -693,9 +693,15 @@ if (CLR_CMAKE_HOST_UNIX)
     endif()
   endif(CLR_CMAKE_HOST_MACCATALYST)
 
+  if(NOT CLR_CROSS_COMPONENTS_BUILD AND CLR_CMAKE_TARGET_ARCH_ARM64 AND CLR_CMAKE_TARGET_LINUX)
```

We should use the host defines instead of the target defines here.

```suggestion
  if(CLR_CMAKE_HOST_ARCH_ARM64 AND CLR_CMAKE_HOST_LINUX)
```

---

Correct, in {build,host,target} scheme, host inherently takes care of cross component bit by the time this script is executed.